script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1D.txt

```diff
@@ -9,10 +9,10 @@ I look up and see that most of the students in the classroom are spending their
 <Page3>
 The teachers who are over thirty say that today's students seem to have no friends in real life.
 Of course, that's an outdated misconception and one that seems unique to people from the previous generation.
-Dependence on the internet and dependence on friends in front of you have become equivalent, and no one in our class is at odds over that.
+Dependence on the internet and dependence on friends in front of you have become equivalent, and no one in our class is at odds over it.
```

Comment:
feel like `that` might be more correct here, as in `and no-one in our class is at odds over that [fact].`